libs/extensions/angular/sidebar-menu/src/components/menu-anchor-item/menu-anchor-item.component.scss

```
@@ -0,0 +1,10 @@
+@use '@kirbydesign/core/src/scss/utils' as utils;
```

Shouldnt be necessary when the partial is already named `utils` 🤞 

```suggestion
@use '@kirbydesign/core/src/scss/utils';
```

---

done